package.json

```diff
@@ -23,7 +22,7 @@
   },
   "devDependencies": {
     "@elastic.io/component-logger": "0.0.1",
-    "chai": "4.2.0",
+    "chai": "^4.2.0",
```

Comment:
We don't use `^` in our package.json files.

Reply:
Done!